<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Simple Transcriber v1.0</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
alternate-background-colo r:  rgb(203, 243, 255);</string>
   </property>
   <property name="text">
    <string>Brouse Files</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SIMPLE TRANSCRIBER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : (2,2,2)
</string>
   </property>
   <property name="text">
    <string>Output Folder</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ONE output file for ALL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color : (0,0,0)
</string>
   </property>
   <property name="text">
    <string>START!</string>
   </property>
  </widget>
  <widget class="QListWidget" name="table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Selected</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
